controllers/controllers.go

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
Any particular reason to have this in `controllers.go`? Seems a little peculiar given everything in here is about controllers.

Reply:
removed.